packages/ui/src/IntervalCard/IntervalCardContainer.tsx

```diff
@@ -0,0 +1,146 @@
+import React from "react";
+
+import { SlotType } from "@eisbuk/shared";
+
+import {
+  IntervalCardVariant,
+  IntervalCardState,
+  IntervalDuration,
+  IntervalCardContainerProps,
+} from "./types";
+
+const IntervalCardContainer: React.FC<IntervalCardContainerProps> = ({
+  type,
+  state,
+  variant,
+  duration,
+  children,
+  className: classes,
+  as = "div",
+}) => {
+  const className = [
+    "relative",
+    ...getContainerSizeClasses(variant, duration),
+    ...getBorderClasses(type, variant, state),
+    getBackgroundColor(type, variant, state),
+    classes,
+  ].join(" ");
+
+  return React.createElement(as, { className }, children);
+};
+
+// #region containerSize
+const getContainerSizeClasses = (
+  variant: IntervalCardVariant,
+  duration: IntervalDuration
+) =>
+  variant !== IntervalCardVariant.Booking
+    ? containerSizeClassesLookup[variant]
+    : bookingContainerSizeLookup[duration];
+
+const containerSizeClassesLookup = {
+  [IntervalCardVariant.Calendar]: [
+    "w-[280px]",
+    "h-[152px]",
+    "rounded-lg",
+    "px-4",
+    "py-3",
+  ],
+  [IntervalCardVariant.Simple]: [
+    "w-[219px]",
+    "h-20",
```

Comment:
Remove explicit height for balanced vertical padding?

Reply:
Left it as is, but you were right...I just replaced the border with an outline and it did wonders.